### PR TITLE
Print error message when initial connection fails

### LINKED ISSUE #432: No error shown when incorrect Slack API Token provided
If an incorrect, or no, Slack API Token is provided in the `plugins.var.python.slack.slack_api_token` option then nothing appears to happen following the `/python reload` cmd.

No error or activity messages are displayed making it hard for the user to troubleshoot the problem.

It would be helpful if an error message could be displayed indicating the missing or incorrect Slack API Token so that the user could resolve the issue.

### PR DESCRIPTION
**Pull request by trygveaa**
I chose to print this instead of using dbg since it's a rather critical
error message.

Fixes #432

### CODE REVIEW

**Comment by selckin** [comment]
+1, one of my slacks stopped connecting without any message, with this applied shows token_revoked

File: wee_slack.py [hunk 1]
```diff
@@ -2103,6 +2102,10 @@ def handle_rtmstart(login_data, eventrouter):
 
         dbg("connected to {}".format(t.domain))
 
+    else:
+        w.prnt("", "ERROR: Failed connecting to Slack with token {}: {}"
+            .format(metadata.token, login_data["error"]))
+
```
Comment (tfheen) [inline wee_slack.py]: Hanging `else` clauses are a bit icky.  I'd much rather have it reverse the check for `login_data["ok"]` and just returning early.  This'll require a reindent, but it's much clearer to read.

Comment (trygveaa) [reply wee_slack.py]: Sure, that's better.

File: wee_slack.py [hunk 2]
```diff
@@ -2031,77 +2031,80 @@ def handle_rtmstart(login_data, eventrouter):
...
+    # try:
+    #    ws = create_connection(web_socket_url, sslopt=sslopt_ca_certs)
+    #    w.hook_fd(ws.sock._sock.fileno(), 1, 0, 0, "receive_ws_callback", t.get_team_hash())
+    #    #ws_hook = w.hook_fd(ws.sock._sock.fileno(), 1, 0, 0, "receive_ws_callback", pickle.dumps(t))
+    #    ws.sock.setblocking(0)
+    #    t.attach_websocket(ws)
+    #    t.set_connected()
+    # except Exception as e:
+    #    dbg("websocket connection error: {}".format(e))
+    #    return False
```
Comment (tfheen) [inline wee_slack.py]: Maybe remove this too, now that you're already making changes here?  Commented-out blocks seems pointless.

Comment (trygveaa) [reply wee_slack.py]: Done.

**Pull request merged**